frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -75,15 +73,13 @@ local symbol_prefix = {
         mem_usage = C_("FooterLetterPrefix", "M:"),
         -- @translators This is the footer letter prefix for Wi-Fi status.
         wifi_status = C_("FooterLetterPrefix", "W:"),
-        -- no prefix for custom text
-        custom_text = "",
     },
     icons = {
         time = "⌚",
         pages_left_book = BD.mirroredUILayout() and "↢" or "↣",
         pages_left = BD.mirroredUILayout() and "⇐" or "⇒",
         battery = "",
-        bookmark_count = "☆",
+        bookmark_count = "\u{F097}", -- "empty bookmark", same as in the bookmark list
```

Comment:
I'm not sure I always wrote it when I used symbols from nerdfont, but I'd really like to see here "from nerdfont" :) just to avoid a round trip to https://www.google.com/search?hl=en&num=25&q=U%2bF097 and wondering wtf.

Reply:
And here?
https://github.com/koreader/koreader/blob/dbebf3f9961a5fdc43fe797939cd66e461c9ed45/frontend/apps/reader/modules/readerbookmark.lua#L29-L31

Reply:
Yep, + `+ en space`.
Are all these just use as prefix for the bookmark item (so, the reason for that EN  SPACE)? Or for other things where they may not be used as prefix (in which case, the EN SPACE may kill some centering if any).
But you can keep all that for a later more bookmark related PR if you want to keep this one focused.